<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWndClass</class>
 <widget class="QMainWindow" name="MainWndClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>406</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MinMaxOpening</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MainWnd.qrc">
    <normaloff>:/MainWnd/Images/Morris.png</normaloff>:/MainWnd/Images/Morris.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="16">
     <widget class="BoardView" name="boardView" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelInput">
      <property name="text">
       <string>Input</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="leInput"/>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="labelOutput">
      <property name="text">
       <string> Output</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="leOutput">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="labelEstimationCount">
      <property name="text">
       <string> # of Estimation</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLineEdit" name="leEstimationCount">
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QLabel" name="labelEstimation">
      <property name="text">
       <string> Estimation</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QLineEdit" name="leEstimation">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="8">
     <widget class="QLabel" name="labelDepth">
      <property name="text">
       <string> Depth</string>
      </property>
     </widget>
    </item>
    <item row="1" column="9">
     <widget class="QLineEdit" name="leDepth">
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="10">
     <widget class="QLabel" name="labelStatus">
      <property name="text">
       <string> Status</string>
      </property>
     </widget>
    </item>
    <item row="1" column="11">
     <widget class="QLineEdit" name="leStatus"/>
    </item>
    <item row="1" column="12">
     <widget class="QLabel" name="labelIdleWhite">
      <property name="text">
       <string>White Idles</string>
      </property>
     </widget>
    </item>
    <item row="1" column="13">
     <widget class="QSpinBox" name="sbIdleWhite">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="maximum">
       <number>9</number>
      </property>
      <property name="value">
       <number>9</number>
      </property>
     </widget>
    </item>
    <item row="1" column="14">
     <widget class="QLabel" name="labelIdleWhite_2">
      <property name="text">
       <string>Black Idles</string>
      </property>
     </widget>
    </item>
    <item row="1" column="15">
     <widget class="QSpinBox" name="sbIdleBlack">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="maximum">
       <number>9</number>
      </property>
      <property name="value">
       <number>9</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Tool bar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionRestart"/>
   <addaction name="actionOpening"/>
   <addaction name="actionGame"/>
   <addaction name="actionStep"/>
   <addaction name="actionRun"/>
   <addaction name="separator"/>
   <addaction name="actionSetting"/>
   <addaction name="actionExit"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="MainWnd.qrc">
     <normaloff>:/MainWnd/Images/open.png</normaloff>:/MainWnd/Images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="MainWnd.qrc">
     <normaloff>:/MainWnd/Images/save.png</normaloff>:/MainWnd/Images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="MainWnd.qrc">
     <normaloff>:/MainWnd/Images/exit.png</normaloff>:/MainWnd/Images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpening">
   <property name="icon">
    <iconset resource="MainWnd.qrc">
     <normaloff>:/MainWnd/Images/opening.png</normaloff>:/MainWnd/Images/opening.png</iconset>
   </property>
   <property name="text">
    <string>Opening</string>
   </property>
  </action>
  <action name="actionGame">
   <property name="icon">
    <iconset resource="MainWnd.qrc">
     <normaloff>:/MainWnd/Images/checkered_flag.png</normaloff>:/MainWnd/Images/checkered_flag.png</iconset>
   </property>
   <property name="text">
    <string>Mid_End</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="icon">
    <iconset resource="MainWnd.qrc">
     <normaloff>:/MainWnd/Images/setting.png</normaloff>:/MainWnd/Images/setting.png</iconset>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="MainWnd.qrc">
     <normaloff>:/MainWnd/Images/question.png</normaloff>:/MainWnd/Images/question.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="icon">
    <iconset resource="MainWnd.qrc">
     <normaloff>:/MainWnd/Images/step.png</normaloff>:/MainWnd/Images/step.png</iconset>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="MainWnd.qrc">
     <normaloff>:/MainWnd/Images/run.png</normaloff>:/MainWnd/Images/run.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="icon">
    <iconset resource="MainWnd.qrc">
     <normaloff>:/MainWnd/Images/restart.png</normaloff>:/MainWnd/Images/restart.png</iconset>
   </property>
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BoardView</class>
   <extends>QWidget</extends>
   <header>BoardView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MainWnd.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWndClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
